tfhe/benches/utilities.rs

```diff
@@ -383,6 +383,29 @@ impl EnvConfig {
     }
 }
 
+/// Generate a number of threads to use to saturate current machine for throughput measurements.
+pub fn throughput_num_threads() -> u64 {
+    let num_threads = rayon::current_num_threads() as f64;
+    // Add 20% more to maximum threads available.
+    (num_threads + (num_threads * 0.2)) as u64
+}
```

Comment:
unclear this is enough, check with @tmontaigu probably

Reply:
Yes its unclear to know if its enough, for some ops it might for others not, or it could be enough for all,

I think a manual check needs to be done by running and trying different values and see if the throughput drastically changes

Reply:
did we do any manual checks ?